proto/redpanda/api/dataplane/v1/topic.proto

```diff
@@ -400,6 +498,68 @@ service TopicService {
       api: API_KAFKA
     };
   }
+  rpc ListTopicsConfigurations(ListTopicsConfigurationsRequest) returns (ListTopicsConfigurationsResponse) {
+    option (google.api.http) = {get: "/v1/topics/configurations"};
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      summary: "List topics with configurations"
+      description: "List topics with their configurations, partition count and replication factor. Optional: filter based on topic name."
+      responses: {
+        key: "200"
+        value: {
+          description: "OK"
+          schema: {
+            json_schema: {ref: ".redpanda.api.dataplane.v1.ListTopicsConfigurationsResponse"}
+          }
+        }
+      }
+    };
+    option (redpanda.api.auth.v1.authorization) = {
+      required_permission: PERMISSION_VIEW
+      api: API_KAFKA
+    };
+  }
+
+  rpc ListLogDirs(ListLogDirsRequest) returns (ListLogDirsResponse) {
+    option (google.api.http) = {get: "/v1/topics/logdirs"};
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      summary: "List log directories"
+      description: "List log directories with size information for topics."
+      responses: {
+        key: "200"
+        value: {
+          description: "OK"
+          schema: {
+            json_schema: {ref: ".redpanda.api.dataplane.v1.ListLogDirsResponse"}
+          }
+        }
+      }
+    };
+    option (redpanda.api.auth.v1.authorization) = {
+      required_permission: PERMISSION_VIEW
+      api: API_KAFKA
+    };
+  }
+
+  rpc ListTopicDocumentations(ListTopicDocumentationsRequest) returns (ListTopicDocumentationsResponse) {
```

Comment:
Do we need this RPC? Seems weird to have a bulk endpoint for this. I think only a per topic endpoint is needed